core/app/modules/import-export/assets/js/pages/import/import-content/import-content.js

```diff
@@ -35,7 +35,7 @@ export default function ImportContent() {
 			return (
 				<ImportContentFooter
 					hasPlugins={ ! ! plugins.length }
-					hasConflicts={ ! ! ( includes.includes( 'templates' ) && uploadedData?.conflicts ) }
+					hasConflicts={ ! ! ( includes.includes( 'templates' ) && uploadedData?.conflicts && Object.keys( uploadedData.conflicts ).length ) }
```

Comment:
```suggestion
					hasConflicts={ ! ! ( includes.includes( 'templates' ) && uploadedData?.conflicts && Object.keys( uploadedData?.conflicts || {} ).length ) }
```

Reply:
Why? I already checked if the conflicts exited.